layouts/_default/profile.html

```diff
@@ -20,28 +20,28 @@
 {{ define "main-class" }}l__profile{{ end }}
 
 {{ define "main" }}
-<div class="container mb-d">
+<div class="container">
     <h1 class="mb-xs">{{.Params.blurb}}</h1>
     <img src="/okf/verein/team-2025.jpeg" style="width: 100%" alt="Teamfoto (eines Teils) der OKF"/>
 
     <h3 class="mt-xs">{{ .Params.intro.title }}</h3>
     {{ .Params.intro.copy | markdownify }}
 </div>
 
-<!--<section class="l__content__bg&#45;&#45;grey l__breakout my-d">-->
-<!--    <div class="container">-->
-<!--        <img src="/okf/verein/spenden.png" alt="Spenden Illustration" width="255">-->
-<!--        <div class="mt-s">-->
-<!--            <h3>{{ .Params.breakout.title }} </h3>-->
-<!--            <p>-->
-<!--                {{ .Params.breakout.copy | markdownify }}-->
-<!--                <a style="display: block" href="../aktivismus-von-innen">-->
-<!--                -> {{ .Params.breakout.link }}-->
-<!--                </a>-->
-<!--            </p>-->
-<!--        </div>-->
-<!--    </div>-->
-<!--</section>-->
+<section class="l__content__bg&#45;&#45;grey l__breakout my-d">
+    <div class="container">
+        <img src="/okf/verein/avi2.jpg" alt="Illustration" width="255">
+        <div class="mt-s">
+            <h3>{{ .Params.breakout.title }} </h3>
+            <p>
+                {{ .Params.breakout.copy | markdownify }}
+                <a style="display: block" href="../aktivismus-von-innen">
+                -> {{ .Params.breakout.link }}
```

Comment:
I'm just now realizing that we usually do it like this: 
```suggestion
                {{ readFile "static/files/icons/arrow_right.svg" | safeHTML }} {{ .Params.breakout.link }}
```